### PR TITLE
More things that we missed in the injections PR

### PR DESCRIPTION
**Pull request by mfojtik**
* Truncate the rm-injections script itself
* Add describer

### CODE REVIEW

**Comment by mfojtik** [comment]
@php-coder FYI, I'll keep adding more here if we find something that we are missing.

**Comment by mfojtik** [comment]
[test]

File: pkg/build/strategies/sti/sti.go [hunk 1]
```diff
@@ -452,11 +452,18 @@ func (b *STI) Execute(command string, user string, config *api.Config) error {
 	// and wait till all injections are uploaded into the container that runs the
 	// assemble script.
 	if len(config.Injections) > 0 && command == api.Assemble {
+		workdir, err := b.docker.GetImageWorkdir(config.BuilderImage)
+		if err != nil {
+			return err
+		}
+		if err := util.FixInjectionsWithRelativePath(workdir, &config.Injections); err != nil {
```
Comment (mfojtik) [inline pkg/build/strategies/sti/sti.go]: @bparees just FYI, I was terribly wrong thinking that the WORKPATH is used automatically, it is not. The default for uploading files into container is set by Docker and it defaults to "/", which is not what we want. This should get the WORKDIR from the builder image and use it as a prefix for relative path...

**Comment by mfojtik** [comment]
[test] (flake)

**Comment by mfojtik** [comment]
@php-coder fixed the `return nil`.

**Comment by php-coder** [comment]
LGTM

**Comment by mfojtik** [comment]
[merge] thanks @php-coder

**Pull request merged**